### DISCUSSION TITLE
Netbox Home Page Error

### DISCUSSION THREAD

**Discussion by gvegapdi** [netbox-community/netbox#11203] [Q&A]
I recently deployed Netbox to Ubuntu Server. I am unable to hit the home page. Sometimes it works but mostly I get the error below. Can anyone help?
![Home Page Error](https://user-images.githubusercontent.com/120682345/207957799-1b5ab212-c388-45dd-ab03-6981dab0f661.png)

**Comment by abhi1693** [comment]
You should take a look at your postgresql server in that case. If this is intermittent error, there is something definitely wrong with it. Maybe not enough resources left for it to run?
